frontends/web/src/api/coins.ts

```diff
@@ -29,3 +30,12 @@ export const subscribeCoinHeaders = (coinCode: CoinCode) => (
     subscribeEndpoint(`coins/${coinCode}/headers/status`, cb)
   )
 );
+
+interface ISetUnit {
+  success: boolean;
+  errorMessage?: string;
+}
```

Comment:
I think it is ok for now, but we should cleanup the different responses.

Ideally we could use SuccessResponse and FailResponse from `api/response.ts`, but it is a bit different :(

Also we have ISuccess from `api/backend.ts` but this optionally contains errorCode, which reminds me shouldn't coins/btc/set-unit also return errorCode?

Reply:
> I think it is ok for now, but we should cleanup the different responses.
Do you mean to uniform the different endpoint responses and errors handling in the `handlers.go` files?

I think in this case an error message could be the right thing, but I don't know if we have a specific ratio to handle error messages vs error codes. @benma ?

Reply:
Modified using `ISuccess`, without error code